arbitrum-docs/stylus/reference/opcode-hostio-pricing.md

```diff
@@ -1,167 +1,171 @@
 ---
-title: 'Opcode and Host I/O Pricing'
-sidebar_label: 'Opcode and Host I/O Pricing'
-description: 'A reference of how much Opcodes and Host I/Os cost in Stylus, with measurements in ink and gas.'
+title: 'Opcode and host I/O pricing'
+sidebar_label: 'Opcode and host I/O pricing'
+description: 'A reference of how much opcodes and host I/Os cost in Stylus, with measurements in ink and gas.'
 author: rachel-bousfield
 sme: rachel-bousfield
 target_audience: 'Developers deploying smart contracts using Stylus.'
 sidebar_position: 4
 ---
 
-This reference provides the latest measurements of WASM opcodes and host I/Os when using Stylus, in both ink and gas. If interested in learning more about the pricing models that Stylus uses to assess the cost of these opcodes and host I/Os, along with its native unit of payment, ink, please visit the [conceptual overview for gas costs in Stylus](https://nitro-docs-git-stylus-gas-offchain-labs.vercel.app/stylus/stylus-gas).
+This reference provides the latest gas and ink costs for specific WASM opcodes and host I/Os when using Stylus. For a conceptual introduction to Stylus gas and ink, see [Gas and ink (Stylus)](/stylus/concepts/stylus-gas).
 
 import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner-partial.md';
 
 <PublicPreviewBannerPartial />
 
-# Opcode table
+## Opcode costs
```

Comment:
intent: align document sections with the language we used to set expectations in intro